Makefile.am

```diff
@@ -44,7 +44,7 @@ usbguard_notifier_SOURCES = \
 	src/NotifyWrapper.cpp \
 	src/Serializer.cpp \
 	src/Log.cpp \
-	icons/usbguard-icon.svg
+	src/usbguard-icon.hpp
```

Comment:
src/NotifyWrapper.cpp is using src/usbguard-icon.hpp header file, so you might need to place this line above src/NotifyWrapper.cpp.